.circleci/config.yml

```diff
@@ -160,6 +177,18 @@ jobs:
       - run: |
           [[ $CIRCLE_TAG == *-* ]] || TELEPRESENCE_VERSION=$CIRCLE_TAG make promote-to-stable
 
+  "release-chart":
+    executor: vm-linux
+    steps:
+    - checkout
+    - install-helm
+    - run: make lint-chart
+    # For now, I don't think we need to build rc charts (nor do I think
+    # we have a need for rc charts), but explicitly calling them out here
+    # in case we change our mind
+    - run: |
+        [[ $CIRCLE_TAG == "*rc*" ]] || BUCKET_DIR="charts" packaging/push_chart.sh
```

Comment:
1. Drop the quotes around `"*rc*"`; because of the double quotes, it'll match `*rc*` literally rather than as a glob.  If this is surprising to you, then it may be helpful to know that despite the visual similarity to the `[` command, the `[[ … ]]` construct is a special piece of Bash syntax, rather than a regular command.
2. I'd be more comfortable excluding _all_ prerelease tags, by having it by `[[ $CIRCLE_TAG == *-* ]]`.

Reply:
Actually: See my next comment below.

Reply:
The `[[ … ]]` check is still broken.